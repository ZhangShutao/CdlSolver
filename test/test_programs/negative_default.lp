% This program is an example of how to implement DI-semantics in cdlp.

a | b.
a :- not &c{not a}, b.
b :- not &c{not b}, a.
a :- b.
:- not b.