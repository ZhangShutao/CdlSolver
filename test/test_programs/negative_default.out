stable model: 1
X:b, a;
A:;
Satisfiable